Xero.Api/XeroApiSettings.cs

```diff
@@ -3,33 +3,33 @@
 
 namespace Xero.Api
 {
-    public class ApplicationSettings
+    public class XeroApiSettings : IXeroApiSettings
     {
         public IConfigurationSection ApiSettings { get; set; }
 
-        public ApplicationSettings(string path)
+        public XeroApiSettings(string path)
         {
             var builder = new ConfigurationBuilder()
                 .AddJsonFile(path)
                 .Build();
 
             ApiSettings = builder.GetSection("XeroApi");
         }
-        public ApplicationSettings() : this("appsettings.json")
+        public XeroApiSettings() : this("appsettings.json")
         {
         }
 
         public string BaseUrl => ApiSettings["BaseUrl"];
 
         public string CallbackUrl => ApiSettings["CallbackUrl"];
 
-        public string Key => ApiSettings["ConsumerKey"];
+        public string ConsumerKey => ApiSettings["ConsumerKey"];
 
-        public string Secret => ApiSettings["ConsumerSecret"];
+        public string ConsumerSecret => ApiSettings["ConsumerSecret"];
 
-        public string SigningCertificatePath => ApiSettings["SigningCertPath"];
+        public string SigningCertificatePath => ApiSettings["SigningCertificatePath"];
```

Comment:
These naming changes will be breaking changes for anyone who already has their appsettings.json files running with the original names right?

Reply:
I guess so, magic strings eh? 

It might be best to leave the strings as they are and simply allow people to implement the new settings Interface with their own KVPs.

Hold on, not sure which version of the code I have but having a IXeroApiSettings (or similar) in the MvcAuthenticator's constructor is the way to go. The version I have does not include it :(

Reply:
Yes, it makes sense to keep the app settings as they were to avoid breaking changes. As @saluton-mundo has said if someone wishes to implement their own app settings they are able to do so.